### ISSUE TITLE
AttributeError: module 'tensorflow' has no attribute 'optimizers'

### ISSUE THREAD

**Issue by alsrgv** [tensorflow/tensorflow#32293]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary): tf-nightly-2.0-preview
- TensorFlow version (use command below):
- Python version: 3.6.8
- Bazel version (if compiling from source): N/A
- GCC/Compiler version (if compiling from source): N/A
- CUDA/cuDNN version: N/A
- GPU model and memory: N/A

**Describe the current behavior**
```
>>> import tensorflow as tf
>>> tf.optimizers
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'tensorflow' has no attribute 'optimizers'
>>> tf.losses
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'tensorflow' has no attribute 'losses'```
```

**Describe the expected behavior**
`tf.optimizers`, `tf.losses` were present.

cc @martinwicke

**Comment by mihaimaruseac** [comment]
I'll make a fix similar to 18c2cf989a2263ee212fbd5ac0b3085d9450b80a, need to check that this works.

Sorry about breakage.

**Comment by mihaimaruseac** [comment]
There is a fix, will land soon and then tonight's nightlies should be ok.
